webpack/assets/javascripts/charts/c3DonutChartService.js

```diff
@@ -0,0 +1,93 @@
+import * as c3 from 'c3';
+
+const enums = {
+    SIZE: {
+        SMALL: {height: 240},
```

Comment:
question: does it make sense to hard code the values or a percentage out of the screen instead? any way to align it to a grid etc?